kafka-ui-api/src/main/java/com/provectus/kafka/ui/config/auth/AbstractAuthSecurityConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.provectus.kafka.ui.config.auth;
+
+public abstract class AbstractAuthSecurityConfig {
```

Comment:
I think ***SecurilyConfig should extend this class to use this constant without full class name specification. Also this class can be made package-private

Reply:
done